src/appsignal/cli.py

```diff
@@ -1,2 +1,118 @@
+from abc import ABC, abstractmethod
+import os
+
+from typing import Optional
+from docopt import docopt
+
+from .__about__ import __version__
+
+DOC = """AppSignal for Python CLI.
+
+Usage:
+  appsignal install [--push-api-key=<key>]
+  appsignal (-h | --help)
+  appsignal --version
+
+Options:
+  -h --help             Show this screen and exit.
+  --version             Show version number and exit.
+  --push-api-key=<key>  Push API Key to use for the installation.
+"""
+
+
 def run():
-    print("Hello AppSignal!")
+    version = f"AppSignal for Python v{__version__}"
+
+    arguments = docopt(DOC, version=version)
+
+    return command_for(arguments).run()
+
+
+class CLICommand(ABC):
+    @abstractmethod
+    def run(self) -> int:
+        raise NotImplementedError
+
+
+def command_for(arguments) -> CLICommand:
+    if arguments["install"]:
+        return InstallCommand(push_api_key=arguments["--push-api-key"])
+    else:
+        raise NotImplementedError
+
+
+INSTALL_FILE_TEMPLATE = """from appsignal import Appsignal
+
+appsignal = Appsignal(
+    active=True,
+    name="{name}",
+    push_api_key="{push_api_key}",
+    environment="development",
```

Comment:
Is there like a `PYTHON_ENV` or other way to dynamically determine the app env?

Reply:
I don't think so. :( All I see online, googling for things like "python production development environment", is people implementing it themselves with some custom environment variable. Django does [something weird](https://docs.djangoproject.com/en/4.2/topics/settings/#envvar-DJANGO_SETTINGS_MODULE) to pick different configurations, which I'm not sure we can derive an environment from.

If it turns out there is a way, we should remove it from the installer-generated file and have the integration itself auto-detect it.

Reply:
> I don't think so. :(

😔 Then we should document that people need to customize that. https://github.com/appsignal/appsignal-docs/issues/1285

> If it turns out there is a way, we should remove it from the installer-generated file and have the integration itself auto-detect it.

Yes, and we can still let users customize it then.

Reply:
Yep, it will need to be done by the user manually. Maybe instead of putting it in the config file, we should have `environment="development"` as one of the config defaults? And then users can override it with `APPSIGNAL_APP_ENV`? Or is it better to have it set here, in the installer? 🤔

Reply:
Development is a good default. If every app has its own way of determine the env, it's up to the devs to configure AppSignal correctly 🤷‍♂️
They can do it just as well in this file with a conditional.

Reply:
Adding it to the default config and leaving it out of the generated file then. 👍

Reply:
I wasn't clear: we can move it to the default config and leave it out of this generated file. We have a similar default in Node.js.